lib/pbench/test/unit/agent/task/test_results_push.py

```diff
@@ -83,6 +86,35 @@ def test_bad_arg():
             > -1
         )
 
+    @staticmethod
+    @responses.activate
+    def test_meta_args():
+        """Test operation when irregular arguments and options are specified"""
+
+        TestResultsPush.add_http_mock_response()
+        runner = CliRunner(mix_stderr=False)
+        result = runner.invoke(
+            main,
+            args=[
+                TestResultsPush.TOKN_SWITCH,
+                TestResultsPush.TOKN_TEXT,
+                TestResultsPush.ACCESS_SWITCH,
+                TestResultsPush.ACCESS_TEXT,
+                TestResultsPush.META_SWITCH,
+                TestResultsPush.META_TEXT_TEST,
+                TestResultsPush.META_TEXT_FOO,
+                tarball,
+            ],
+        )
+        assert result.exit_code == 2
+        assert (
+            result.stderr.find(
+                "Invalid value for 'RESULT_TB_NAME': "
+                "File 'pbench.sat:FOO' does not exist."
+            )
+            > -1
+        )
```

Comment:
Rather than using `find()` with a comparison against `-1`, how about using `in`?
```
        assert (
            "Invalid value for 'RESULT_TB_NAME': "
            "File 'pbench.sat:FOO' does not exist."
            in result.stderr
        )
```
(If you get complaints about mismatches between bytes-like objects and strings, I think there is an accessor for `result` which returns `stderr` a proper string, or you can always declare the comparison to be a bytes literal by putting a `b` in front of the opening quote.)